src/components/BottomNavBar.tsx

```diff
@@ -0,0 +1,57 @@
+import { Flex, Text, Center, Avatar } from '@chakra-ui/react';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { ROUTES } from '../constants/routes';
+
+function BottomNavBar() {
+  const [isLogin] = useState(1); // 임시로 (네브바 : 로그인 버튼) 중 뭐가 보일지 결정하는 상태입니다
+
+  return (
+    <>
+      <Flex
+        position="fixed"
+        w="100%"
+        padding="10px"
+        color="white"
+        textAlign="center"
+        bottom="0"
+        left="0"
+        right="0"
+        cursor="pointer"
+        backgroundColor={isLogin ? '#ECE9E9' : '#28B67E'}
+        alignItems="center"
+        justifyContent="center"
```

Comment:
```suggestion
        align="center"
        justify="center"
```
<img width="679" alt="image" src="https://github.com/prgrms-fe-devcourse/FEDC4_Campers_Jaeho/assets/66409882/7b6d9e7d-6973-4e9e-b1fd-813d858d53b1">
chakra 공식문서에 보면 위와 같이 간략하게 사용할 수 있습니다.